variants/uPesy_esp32c3_basic/pins_arduino.h

```diff
@@ -0,0 +1,36 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+
+#define USB_VID 0x303A
+#define USB_PID 0x8195
+#define USB_MANUFACTURER "uPesy Electronics"
+#define USB_PRODUCT "uPesy ESP32C3 Basic"
+#define USB_SERIAL ""
+
+#define EXTERNAL_NUM_INTERRUPTS 22
+#define NUM_DIGITAL_PINS        22
+#define NUM_ANALOG_INPUTS       6
+
```

Comment:
Those #defines are no longer needed in pins_arduino.h file. They are provided globally per SOC.
```suggestion
```